src/Update/Nix/FetchGit/Prefetch.hs

```diff
@@ -20,25 +20,20 @@ import           Update.Nix.FetchGit.Warning
 data NixPrefetchGitOutput = NixPrefetchGitOutput{ url    :: Text
                                                 , rev    :: Text
                                                 , sha256 :: Text
+                                                , date   :: Text
                                                 }
   deriving (Show, Generic, FromJSON)
 
 -- | Run nix-prefetch-git
 nixPrefetchGit :: Text -- ^ The URL to prefetch
                -> IO (Either Warning NixPrefetchGitOutput)
 nixPrefetchGit prefetchURL = do
-  let nixPrefetchCommand = "nix-prefetch-git " ++ unpack prefetchURL
-      nixShellArgs = [ "-p", "nix-prefetch-git"
-                     , "-p", "nix"
-                     , "--command", nixPrefetchCommand
-                     ]
-      nsStdin = ""
   (exitCode, nsStdout, nsStderr) <-
-    readProcessWithExitCode "nix-shell" nixShellArgs nsStdin
+    readProcessWithExitCode "nix-prefetch-git" [unpack prefetchURL] ""
```

Comment:
What's the reason behind not using nix-shell here? I suppose the argument could be made that this should always be run in an environment with nix-prefetch-git in.